### PR TITLE
solved undo bug

### PR DESCRIPTION
**Pull request by Manvitha-Vema**
i have solved the issue

### CODE REVIEW

File: fingerpaint.js [hunk 1]
```diff
@@ -73,66 +207,43 @@ window.onload=function()
...
-            context.moveTo(e.targetTouches[0].clientX,e.targetTouches[0].clientY);
-        } else
-        {
-            context.moveTo(e.targetTouches[0].clientX,e.targetTouches[0].clientY);
-        }
-    }else{
-        if(drawing)
-        {
-           
+function handleMouseMove(e) {
```
Comment (KingSlayr) [inline fingerpaint.js:217]: Are you solving cursor going out of the canvas issue?

File: fingerpaint.js [hunk 2]
```diff
@@ -1,70 +1,204 @@
+// var drawing = false;
+// var context;
```
Comment (KingSlayr) [inline fingerpaint.js:2]: Reason to comment out this code?

File: fingerpaint.js (hunk 1 above)
Comment (Manvitha-Vema) [reply fingerpaint.js:217]: no i have just solved undo bug

File: fingerpaint.js (hunk 2 above)
Comment (Manvitha-Vema) [reply fingerpaint.js:2]: sorry i forgot to delete it.
it was all same except the undo

File: fingerpaint.js (hunk 1 above)
Comment (KingSlayr) [reply fingerpaint.js:217]: Changes seems to be in mouse events handler function , can you please explain the solution?

Comment (Manvitha-Vema) [reply fingerpaint.js:217]: i have tried to do the mouse one,so i simplified it

Comment (KingSlayr) [reply fingerpaint.js:217]: Is it working and is the bug solved?

Comment (KingSlayr) [reply fingerpaint.js:217]: Can you please deploy the website on GitHub pages on your fork and share link for testing?

Comment (Manvitha-Vema) [reply fingerpaint.js:217]: undo was fine,the cursor isn't fixed

Comment (KingSlayr) [reply fingerpaint.js:217]: Ok, I'm not able to understand the solution 👀

Comment (Manvitha-Vema) [reply fingerpaint.js:217]: i have simplified the logic for touch and mouse events by solely relying on lineTo and stroke but implemented no movement is done when the mouse is not pressed.

Comment (KingSlayr) [reply fingerpaint.js:217]: Can you share a deployed link?

Comment (Manvitha-Vema) [reply fingerpaint.js:217]: I've just ran that on my local system. Do you want me to deploy it now and share.

**Comment by KingSlayr** [comment]
Code not acceptable

**Pull request closed without merging**